src/Pos/Txp/MemState/Class.hs

```diff
@@ -18,9 +21,7 @@ module Pos.Txp.MemState.Class
        ) where
 
 import qualified Control.Concurrent.STM as STM
-import           Control.Monad.Except   (ExceptT)
-import           Control.Monad.State    (StateT)
-import           Control.Monad.Trans    (MonadTrans)
+import qualified Control.Monad.Ether    as Ether.E
```

Comment:
it's a shorthand for `Ether.Explicit`, since `Explicit` is a separate word I separate it by a dot, which is the usual convention of module naming (e.g. `Control.Monad`, not `ControlMonad`).